index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
> [T]he longer we leave [this change] the more disruptive it is for our users when we do pull the plug.

It seems the other way around to me. In a year, say, the number of people whose browsers don't provide `String#trim` will be lower than it is today, presumably.

I don't feel strongly about this change, though, so if others are in favour I don't mind merging it.